prbt_support/urdf/prbt_macro.xacro

```diff
@@ -229,7 +233,8 @@ limitations under the License.
     <link name="${prefix}link_5">
```

Comment:
What about the flange? It is still in the representation in which we state the inertia with regard to the link frame. Is this different representation ok?